service-alerts-connector/service-alerts-api.yaml

```diff
@@ -0,0 +1,303 @@
+  /v1/coct-service_alerts-7days-unplanned.json:
+    get:
+      tags:
+        - "v1"
+      summary: Unplanned outages for Last 7 Days
+      description: Get all unplanned service outages active over the last 7 days with social media fields
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV0'
+  /v1/coct-service_alerts-7days-planned.json:
+    get:
+      tags:
+        - "v1"
+      summary: Planned Outages for Last 7 Days
+      description: Get all planned service outages active in the last 7 days with social media fields
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV0'
+  /v1/coct-service_alerts-all-unplanned.json:
+    get:
+      tags:
+        - "v1"
+      summary: All unplanned outages
+      description: Get all unplanned service outages going back several years with social media fields.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV1'
+  /v1/coct-service_alerts-all-planned.json:
+    get:
+      tags:
+        - "v1"
+      summary: All planned outages
+      description: Get all planned service outages going back several years with social media fields.
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ServiceAlertV1'
+  /v1/alerts/{alert_id}.json:
+    get:
+      tags:
+        - "v1"
+      summary: Get a specific service alert by ID
+      description: Retrieve detailed information about a service alert by its unique ID
+      parameters:
+        - name: alert_id
+          in: path
+          description: ID of the service alert to retrieve
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ServiceAlertV1'
+components:
+  schemas:
+    ServiceAlertV0:
+      type: object
+      properties:
+        Id:
+          type: integer
+          description: Unique identifier for service alert
+          example: 23128
+        service_area:
+          type: string
+          description: Type of service affected
+          example: "Water & Sanitation"
```

Comment:
It would be really nice if this could be modeled (in V2) as an enum. Anyone using this data will effectively be doing this in their code anyway, and it would be nice to have the full list of possible values available in the spec

Reply:
Let me double check that isn't an enum already in the form that is used to create the observations. If it is, I'll update the `v0` and `v1` path specs, if not, I'll create an issue for it to go into `v2`

Reply:
Ah, it is indeed already an enum - will update the PR. Thanks for the comment pointing us in this direction!

Reply:
OK, interestingly enough, turns out `title` is also an eum in the original form, but with hundreds of entries.

In the interests of not overloading the spec, I'm inclined to not make it an enum.